src/components/Comments.jsx

```diff
@@ -0,0 +1,39 @@
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import { getCommentsByReview } from "../api";
+
+function Comments() {
+  const [commentClick, setCommentClick] = useState();
+  let { review_id } = useParams();
+  console.log(review_id);
+
+  useEffect(() => {
+    getCommentsByReview(review_id).then((results) => {
+      setCommentClick(results.review);
+      console.log(commentClick);
+    });
+  }, []);
```

Comment:
Line 15, useEffect dependency should have review_id in there so that when we go to another review, we fetch that review's comments
The useState for commentClick is a bit misleading as well, from my understanding this is where you are storing your review comments but it's also being used as a kind of "loading state" currently (if no comments, display loading otherwise display comments)
It would be better to separate these and have your initial useState as an empty array for your comment list, and a separate loading useState with a boolean as you have done for other components